packages/core/package.json

```diff
@@ -31,30 +31,18 @@
     "lib"
   ],
   "dependencies": {
-    "@chainsafe/libp2p-noise": "7.0.0",
-    "@hoprnet/hopr-connect": "workspace:packages/connect",
     "@hoprnet/hopr-core-ethereum": "workspace:packages/core-ethereum",
     "@hoprnet/hopr-utils": "workspace:packages/utils",
     "@libp2p/interface-peer-id": "1.0.5",
     "@libp2p/interfaces": "2.0.4",
-    "@libp2p/kad-dht": "1.0.16",
-    "@libp2p/mplex": "1.2.2",
     "@libp2p/peer-id": "1.1.16",
-    "@libp2p/peer-id-factory": "1.0.18",
     "@multiformats/multiaddr": "10.3.3",
     "bn.js": "5.2.1",
-    "datastore-level": "^8.0.0",
     "debug": "4.3.4",
     "heap-js": "2.2.0",
-    "it-pushable": "3.0.0",
-    "libp2p": "0.37.3",
     "retimer": "3.0.0"
```

Comment:
Is retimer still used somewhere in the core?

Reply:
Yes, for triggering the strategy tick